src/browser-sandbox/pruneNonCriticalSelectors.js

```diff
@@ -90,12 +90,17 @@ export default function pruneNonCriticalSelectors ({
     )
   }
 
-  // give some time (renderWaitTime) for sites like facebook that build their page dynamically,
-  // otherwise we can miss some selectors (and therefor rules)
-  // --tradeoff here: if site is too slow with dynamic content,
-  // it doesn't deserve to be in critical path.
-  return sleep(renderWaitTime).then(() => {
-    console.log('debug: waited for renderWaitTime: ' + renderWaitTime)
-    return filterSelectors(selectors)
-  })
+  try {
```

Comment:
This added try/carch should not be necessary; the call of this function is already wrapped in a try/catch in core.js.